sale_cutoff_time_delivery/models/cutoff_time_mixin.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class CutoffTimeMixin(models.AbstractModel):
+
+    _name = "cutoff.time.mixin"
```

Comment:
Prefer sale.cutoff.time.mixin as this is applicable to sale. Either, externalize in another module